.github/workflows/main-ci.yml

```diff
@@ -111,7 +111,7 @@ jobs:
     strategy:
       matrix:
         platform: [arm64-darwin, x86_64-ios]
-    runs-on: ubuntu-20.04
+    runs-on: macOS-latest
```

Comment:
We have issues with frameworks linking if build on ubuntu.
Here is diff in `otool -L ` between Mac and ubuntu:

```
Mac:
	/System/Library/Frameworks/CoreFoundation.framework/CoreFoundation (compatibility version 150.0.0, current version 1856.105.0, weak)

Ubuntu:
	/System/Library/Frameworks/AVFAudio.framework/AVFAudio (compatibility version 1.0.0, current version 1.0.0)
	/System/Library/Frameworks/CoreFoundation.framework/CoreFoundation (compatibility version 150.0.0, current version 1856.105.0)

```